base-images/utils/aipcc.sh

```
@@ -1,7 +1,331 @@
 #!/usr/bin/env bash
 set -Eeuxo pipefail
 
-DNF_OPTS=(-y --nodocs --setopt=install_weak_deps=False --setopt=keepcache=True)
+ARCH=${TARGETARCH}
```

Shellcheck (0.11.0)

[warning] ARCH appears unused. Verify use (or export if used externally).

(SC2034)